src/systems/velocity_control/VelocityControl.cc

```diff
@@ -86,15 +114,43 @@ void VelocityControl::Configure(const Entity &_entity,
     return;
   }
 
-  // Subscribe to commands
-  std::string topic{"/model/" + this->dataPtr->model.Name(_ecm) + "/cmd_vel"};
+  // Subscribe to model commands
+  std::string modelTopic{"/model/" + this->dataPtr->model.Name(_ecm) + "/cmd_vel"};
   if (_sdf->HasElement("topic"))
-    topic = _sdf->Get<std::string>("topic");
+    modelTopic = _sdf->Get<std::string>("topic");
+  modelTopic = transport::TopicUtils::AsValidTopic(modelTopic);
```

Comment:
you should check if the string is null

Reply:
added check. 468e2f68383b9188c10b1201fda73ac863eb8a42